src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/DefaultMotion.tsx

```diff
@@ -36,7 +36,7 @@ const DefaultMotion = ({
     return null;
   }
 
-  const motionState = getMotionState(networkMotionState, actionData.motionData);
+  const motionState = MotionState.Voting; //getMotionState(networkMotionState, actionData.motionData);
```

Comment:
just testing the ui in this pr, see https://github.com/JoinColony/colonyCDapp/pull/380 for wiring